model/system/deprecated/registry-deprecated.yaml

```diff
@@ -51,3 +51,17 @@ groups:
         stability: experimental
         deprecated: 'Replaced by `cpu.mode`'
         examples: [ "idle", "interrupt" ]
+
+  - id: registry.system.metrics.deprecated
+    type: attribute_group
+    display_name: Deprecated System Metrics
+    brief: >
+      "Describes deprecated system metrics attributes."
+    stability: experimental
+    attributes:
```

Comment:
```suggestion
```

just add it  to the existing group - there is in special meaning in the group name and properties here -  it does not matter which `attribute_group` attribute is declared in for tooling